Explorer/Assets/DCL/WorldTime/IWorldTimeProvider.cs

```diff
@@ -0,0 +1,14 @@
+using Cysharp.Threading.Tasks;
+using System;
+using System.Threading;
+
+namespace DCL.Time
+{
+    public interface IWorldTimeProvider : IDisposable
+    {
+        public UniTask<float> GetWorldTimeAsync(CancellationToken cancellationToken);
+
+        //Not used yet, but in old renderer was used by skybox controller, so there might be an use case
+        public void SetPausedState(bool isPaused);
```

Comment:
Do we plan to use it in the short-term? Lets delete it if its not.

Reply:
good question xD I will remove it for now and next week we will discuss with Misha what we wanna do about the skybox